### DISCUSSION TITLE
Issue while trying to download https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.51.0

### DISCUSSION THREAD

**Discussion by LFCunha10** [Homebrew/discussions#3951] [Everyday usage]
Hello.

I'm trying to install MongoDB on my Mac M1, using `brew install mongodb-community@6.0`. However, I'm facing the following error
`mongodb-community@6.0: Failed to download resource "libnghttp2_bottle_manifest"`
`Download failed: https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.51.0`

How can I solve this?

Thanks

**Comment by SMillerDev** [comment]
See https://github.com/orgs/Homebrew/discussions/3952

**Reply by amartinwest** [reply]
Seems to be fixed now, hit it trying to download php
